### PR TITLE
tmz 2.1.0 bugfix ccitt punchtape

### PR DESCRIPTION
**Pull request by t-m-z**
refactor punchtape
refactor ccitt
correct symboltables
add ccitt codes mtk2, ustty

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -3841,10 +3864,19 @@
...
-  "symboltables_murraybaudot_title": "Murray Baudot Code",
+  "symboltables_murraybaudot_title": "Murray Baudot Code, ITA2",
   "symboltables_murraybaudot_description": "Teletype code, CCITT-2 (Punched Cards) by Donald Murray",
   "symboltables_murraybaudot_shift_to_sign_table": "NUMERALS FOLLOW",
   "symboltables_murraybaudot_shift_to_letter_table": "LETTERS FOLLOW",
+  "symboltables_murraybaudot_bell": "BELL",
+  "symboltables_murraybaudot_whoareyou": "Who are you",
+  "symboltables_murraybaudot_carriagereturn": "Carriage return",
+  "symboltables_murraybaudot_linefeed": "Linefeed",
+  "symboltables_murraybaudot_miss_title": "Murray Baudot Code, miss interpreted",
```
Comment (S-Man42) [inline assets/i18n/en.json:3875]: misinterpreted

File: assets/searchstrings/en.json [hunk 2]
```diff
@@ -63,8 +63,6 @@
   "catan": "thesettlerofcatan tabletopgame boardgame",
   "callforcontribution": "contributions contribute",
   "ccitt": "telegraph punchedtapes punchtapes punchedpapertapes cards",
```
Comment (S-Man42) [inline assets/searchstrings/en.json:65]: please add papertapes

Comment (S-Man42) [reply assets/searchstrings/en.json:65]: and pipelinings

**Comment by t-m-z** [comment]
all done

**Pull request merged**